### PR TITLE
Make v2 bytes value displayed same as v1 hex string

### PR DESCRIPTION
**Pull request by xiangfu0**
Make v2 bytes value displayed same as v1 hex string.
image
image

### CODE REVIEW

**Review by walterddr — COMMENTED** [review]
can we add a test?
the util is testable via runReducer method

**Comment by xiangfu0** [comment]
> can we add a test? the util is testable via runReducer method

Existing h2 queries cover here. Hence I need to make the change on the comparator.

File: pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTestBase.java [hunk 1]
```diff
@@ -181,6 +182,9 @@ protected void compareRowEquals(List<Object[]> resultRows, List<Object[]> expect
         }
         return Double.compare(ld, rd);
       } else if (l instanceof String) {
+        if (r instanceof byte[]) {
```
Comment (xiangfu0) [inline pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTestBase.java:185]: When querying bytes column, left side from pinot is a hex string, right side from h2 is byte[]

Comment (walterddr) [reply pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTestBase.java:185]: ah. got it... so we want to make it hex string. that sound like a good plan

Comment (xiangfu0) [reply pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTestBase.java:185]: postgres give hex value, but with with a prefix: `\x`, e.g. `\x4A6F686E20446F65`

**Review by walterddr — APPROVED** [review]

**Pull request merged**